### PR TITLE
[zh]Sync reference-1

### PR DESCRIPTION
**Pull request by nameYULI**

### CODE REVIEW

**Comment by nameYULI** [comment]
related issue:https://github.com/kubernetes/website/issues/33478
reference-1
`
1	1	content/en/docs/reference/kubernetes-api/common-parameters/common-parameters.md
1	3	content/en/docs/reference/kubernetes-api/common-definitions/list-meta.md
7	7	content/en/docs/reference/kubernetes-api/common-definitions/object-meta.md
96	0	content/en/docs/reference/labels-annotations-taints/_index.md
`

File: content/zh/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -1248,8 +1289,123 @@ See more details on the [Audit Annotations](/docs/reference/labels-annotations-t
...
+Example: `node-role.kubernetes.io/control-plane:NoSchedule`
+-->
+例子：`node-role.kubernetes.io/control-plane:NoSchedule`
+
+用于：Node
+
+<!--
+Taint that kubeadm applies on control plane nodes to allow only critical workloads to schedule on them.
+-->
+kubeadm 应用在控制平面节点上以仅允许在其上安排关键工作负载的污点。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/_index.md:1391]:
```suggestion
kubeadm 应用在控制平面节点上的污点，以仅允许在其上安排关键工作负载。
```

File: content/zh/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -1248,8 +1289,123 @@ See more details on the [Audit Annotations](/docs/reference/labels-annotations-t
...
+-->
+例子：`node-role.kubernetes.io/master:NoSchedule`
+
+用于：Node
+
+<!--
+Taint that kubeadm applies on control plane nodes to allow only critical workloads to schedule on them.
+Starting in v1.20, this taint is deprecated in favor of `node-role.kubernetes.io/control-plane` and will be removed in v1.25.
+-->
+kubeadm 应用在控制平面节点上以仅允许在其上安排关键工作负载的污点。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/_index.md:1408]:
```suggestion
kubeadm 应用在控制平面节点上的污点，以仅允许在其上安排关键工作负载。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-meta.md [hunk 3]
```diff
@@ -390,11 +390,12 @@ ObjectMeta 是所有持久化资源必须具有的元数据，其中包括用户
...
-  The name of the cluster which the object belongs to. This is used to distinguish resources with same name and namespace in different clusters. This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request.
-  -->
-  对象所属的集群的名称。这用于区分不同集群中具有相同名称和命名空间的资源。
-  该字段现在没有在任何地方设置，如果在创建或更新请求中设置，apiserver 将忽略它。
+<!-- 
+Deprecated: ClusterName is a legacy field that was always cleared by the system and never used; it will be removed completely in 1.25.
+The name in the go struct is changed to help clients detect accidental use.
+-->
+已弃用：ClusterName 是一个总是被系统清除并且从未使用过的遗留字段；它将在 1.25 中完全删除。
+go 结构体中的名称已更改，以帮助客户端检测意外使用。
```
[the remark this answers is not in the document]
Comment (nameYULI) [reply content/zh/docs/reference/kubernetes-api/common-definitions/object-meta.md]: thanks a lot for your guidance

File: content/zh/docs/reference/kubernetes-api/common-parameters/common-parameters.md [hunk 4]
```diff
@@ -86,6 +86,32 @@ A selector to restrict the list of returned objects by their fields. Defaults to
...
+fieldValidation instructs the server on how to handle objects in the request (POST/PUT/PATCH) containing unknown or duplicate fields, provided that the `ServerSideFieldValidation` feature gate is also enabled.
+-->
+fieldValidation 指示服务器如何处理请求（POST/PUT/PATCH）中包含未知或重复字段的对象，
+前提是 `ServerSideFieldValidation` 特性门控也已启用。
+<!--
+Valid values are: - Ignore: This will ignore any unknown fields that are silently dropped from the object, and will ignore all but the last duplicate field that the decoder encounters. This is the default behavior prior to v1.23 and is the default behavior when the `ServerSideFieldValidation` feature gate is disabled.
+-->
+有效值为：
+- Ignore：这将忽略从对象中默默删除的所有未知字段，并将忽略除解码器遇到的最后一个重复字段之外的所有字段。
+这是在 v1.23 之前的默认行为，也是当 `ServerSideFieldValidation` 特性门控被禁用时的默认行为。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-parameters/common-parameters.md:101]: 这是一个无序列表，条目内所有文字，包括注释，都按第一行缩进。下同。

```suggestion
  这是在 v1.23 之前的默认行为，也是当 `ServerSideFieldValidation` 特性门控被禁用时的默认行为。
```

File: content/zh/docs/reference/labels-annotations-taints/_index.md [hunk 5]
```diff
@@ -1008,6 +1008,29 @@ kubelet 检查 `/proc/sys/kernel/pid_max` 大小的 D 值和 Kubernetes 在 Node
...
 然后该指标与在 kubelet 上设置的相应阈值进行比较，以确定是否应该添加/删除 Node 状况和污点。
 
+### node.kubernetes.io/out-of-service
+<!--
+Example: `node.kubernetes.io/out-of-service:NoExecute`
+A user can manually add the taint to a Node marking it out-of-service. If the `NodeOutOfServiceVolumeDetach` 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled on
+`kube-controller-manager`, and a Node is marked out-of-service with this taint, the pods on the node will be forcefully deleted if there are no matching tolerations on it and volume detach operations for the pods terminating on the node will happen immediately. This allows the Pods on the out-of-service node to recover quickly on a different node.
+-->
+例子：`node.kubernetes.io/out-of-service:NoExecute`
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/_index.md:1018]: 此行后有一空行，用来划分段落，不可删除。

File: content/zh/docs/reference/labels-annotations-taints/_index.md [hunk 6]
```diff
@@ -1233,11 +1256,34 @@ based on setting `securityContext` within the Pod's `.spec`.
...
+Value can either be `true` or `false`.
+This determines whether a user can modify the mode of the source volume when a
+{{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}} is being created from a VolumeSnapshot.
+Refer to [Converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode) and the [Kubernetes CSI Developer Documentation](https://kubernetes-csi.github.io/docs/) for more information.
+-->
+值可以是 `true` 或者 `false`。
+这决定了当从 VolumeSnapshot 创建 {{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}}
+时，用户是否可以修改源卷的模式。
+更多信息请参阅[转换快照的卷模式](/zh/docs/concepts/storage/volume-snapshots/#convert-volume-mode)和
+[Kubernetes CSI 开发者文档](https://kubernetes-csi.github.io/docs/)
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/_index.md:1278]:
```suggestion
[Kubernetes CSI 开发者文档](https://kubernetes-csi.github.io/docs/)。
```

File: content/zh/docs/reference/labels-annotations-taints/_index.md [hunk 7]
```diff
@@ -1248,8 +1294,124 @@ See more details on the [Audit Annotations](/docs/reference/labels-annotations-t
...
+Example: `node-role.kubernetes.io/control-plane:NoSchedule`
+-->
+例子：`node-role.kubernetes.io/control-plane:NoSchedule`
+
+用于：Node
+
+<!--
+Taint that kubeadm applies on control plane nodes to allow only critical workloads to schedule on them.
+-->
+kubeadm 应用在控制平面节点上的污点，以仅允许在其上安排关键工作负载。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/_index.md:1397]:
```suggestion
kubeadm 应用在控制平面节点上的污点，仅允许在其上调度关键工作负载。
```

File: content/zh/docs/reference/labels-annotations-taints/_index.md [hunk 8]
```diff
@@ -1248,8 +1294,124 @@ See more details on the [Audit Annotations](/docs/reference/labels-annotations-t
...
+-->
+例子：`node-role.kubernetes.io/master:NoSchedule`
+
+用于：Node
+
+<!--
+Taint that kubeadm applies on control plane nodes to allow only critical workloads to schedule on them.
+Starting in v1.20, this taint is deprecated in favor of `node-role.kubernetes.io/control-plane` and will be removed in v1.25.
+-->
+kubeadm 应用在控制平面节点上的污点，以仅允许在其上安排关键工作负载。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/_index.md:1414]:
```suggestion
kubeadm 应用在控制平面节点上的污点，仅允许在其上调度关键工作负载。
```

File: content/zh/docs/reference/labels-annotations-taints/_index.md [hunk 9]
```diff
@@ -1248,8 +1294,124 @@ See more details on the [Audit Annotations](/docs/reference/labels-annotations-t
...
+{{< note >}}
+从 v1.20 开始，此污点已弃用，取而代之的是 `node-role.kubernetes.io/control-plane`，并将在 v1.25 中将其删除。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/_index.md:1416]:
```suggestion
从 v1.20 开始，此污点已弃用，并将在 v1.25 中将其删除，取而代之的是 `node-role.kubernetes.io/control-plane`。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-meta.md [hunk 10]
```diff
@@ -391,10 +391,11 @@ ObjectMeta 是所有持久化资源必须具有的元数据，其中包括用户
 - **clusterName** (string)
 
   <!-- 
-  The name of the cluster which the object belongs to. This is used to distinguish resources with same name and namespace in different clusters. This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request.
+  Deprecated: ClusterName is a legacy field that was always cleared by the system and never used; it will be removed completely in 1.25.
+  The name in the go struct is changed to help clients detect accidental use.
   -->
-  对象所属的集群的名称。这用于区分不同集群中具有相同名称和命名空间的资源。
-  该字段现在没有在任何地方设置，如果在创建或更新请求中设置，apiserver 将忽略它。
+  已弃用：ClusterName 是一个总是被系统清除并且从未使用过的遗留字段；它将在 1.25 中完全删除。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-meta.md:397]:
```suggestion
  已弃用：clusterName 是一个总是被系统清除并且从未使用过的遗留字段；它将在 1.25 中完全删除。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-meta.md [hunk 11]
```diff
@@ -391,10 +391,11 @@ ObjectMeta 是所有持久化资源必须具有的元数据，其中包括用户
...
+  go 结构体中的名称已更改，以帮助客户端检测意外使用。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-meta.md:398]:
```suggestion
  go 结构体中的对应字段名称已更改，以帮助客户端检测意外使用。
```

**Pull request merged**